docs/generated.d.ts

```diff
@@ -475,7 +475,7 @@ export type AbstractTransaction = BaseTransaction & {
   /**
    * List of transaction events
    */
-  events: TransactionEvent[];
```

Comment:
This would make the tx response object a bit annoying to use for endpoints that are known to include the events property. A lot of client code would have to perform null/undefined checks on the property before use.

Would it make sense to have something like a "TransactionWithEvents" schema/type where this isn't optional, and specify that as the return type for endpoints like `/extendex/v1/tx/<txid>`?